webview/src/plots/util/messages.ts

```diff
@@ -3,16 +3,15 @@ import { PlotsSection } from 'dvc/src/plots/webview/contract'
 import { sendMessage } from '../../shared/vscode'
 import { PlotGroup } from '../components/templatePlots/templatePlotsSlice'
 
-export const addCustomPlot = () =>
+export const addPlot = () =>
   sendMessage({
-    type: MessageFromWebviewType.ADD_CUSTOM_PLOT
+    type: MessageFromWebviewType.ADD_PLOT
   })
 
-export const addPipelinePlot = () => {
+export const addCustomPlot = () =>
```

Comment:
I chose to keep the add plot button in the custom section since we still have the trash icon there, so I thought it made sense to have the add button next to it. 

<img width="1368" alt="image" src="https://github.com/iterative/vscode-dvc/assets/43496356/41335289-aafa-4db3-9cde-d01f63c67eb0">

But we can remove it if preferred.

Reply:
If you think the UX is better then it is ok to leave and we can even keep the add plot button above the `Data Series` section but do keep in mind that we will to maintain this code. Any code that runs can eventually cause bugs.

Reply:
Are we still keeping this?

Reply:
I thought we agreed to keep it for now, but it wouldn't be the first time I misunderstood what was said in a meeting 😊. I'll go ahead and take off the `+` button. The `addCustomPlot` util would stay since we use it in the error screen.

Reply:
Actually, after testing it, we should keep it for now until we add a button to our "No Plots Added" screen.